extensions/wikia/UserPreferencesV2/UserPreferencesV2.class.php

```diff
@@ -175,11 +175,6 @@ static public function onGetPreferences( $user, &$defaultPreferences ) {
 			}
 		}
 
-		if ( isset( $defaultPreferences['htmlemails'] ) ) {
```

Comment:
Yep, the whole statement should be removed.

We had been adding 'htmlemails' to the defaultPreferences array [here](https://github.com/Wikia/app/pull/6950/files#diff-bf6d5d0093a049e88d2096e1be377e07L464), and then this code was called as part of a hook to actually add the checkbox. Now that we're deleting that code from Preferences.php, if we remove this check it this will throw an error since there's no longer an `htmlemails` key. We also just want to get rid of the option altogether, so deleting is the way to go :)